db/migrate/20160927230134_disable_fk_message_digests_nodes.rb

```diff
@@ -0,0 +1,21 @@
+class DisableFkMessageDigestsNodes < ActiveRecord::Migration
+
+  class Bag < ActiveRecord::Base
+  end
+
+  class MessageDigest < ActiveRecord::Base
+    belongs_to :bag
+  end
+
+  def up
+    remove_foreign_key :message_digests, :nodes
+  end
+
+  def down
+    add_foreign_key :message_digests, :nodes,
+      name: 'message_digests_nodes_id_fk',
```

Comment:
The migration has failed before because of conflicts in names, so this was added to fix that and it will be more robust and more meaningful with the name.